docs/install/building.rst

```diff
@@ -27,3 +27,24 @@ available to turn on debug builds, address sanitizer, etc.).
 * :repo:`CMakeLists.txt`
 * :repo:`common.cmake`
 * :repo:`thirdparty.cmake`
+
+(Experimental) Building with Bazel_
+
+Follow the instruction_ to install Bazel, then run following command to build Envoy:
+
+.. code-block:: console
+
+  bazel build //:envoy
+
+The built binary will be at `bazel-bin/envoy`
+
+To run tests, run following command:
+
+.. code-block:: console
+
+  bazel test //:envoy-test
+
+Note not all tests pass with Bazel yet.
```

Comment:
bazel test run tests in different paths than do_ci.sh does, results some fixtures cannot be found. I will do a follow up PR after this to fix them.